src/payment/strategies/ppsdk/ppsdk-sub-strategy.ts

```diff
@@ -8,5 +9,9 @@ export interface SubStrategySettings {
 }
 
 export interface SubStrategy {
-    execute(settings: SubStrategySettings): Promise<void>;
+    execute(settings: SubStrategySettings): void;
```

Comment:
Are you sure you want to change from `Promise<void>` to `void`? I can see that the implementation actually treats this as an async method, but the interface doesn't seem to reflect that. e.g.: https://github.com/bigcommerce/checkout-sdk-js/blob/9123b21a61f2b9d69d38a6bbed99d6dd19c3f962/src/payment/strategies/ppsdk/sub-strategies/card-sub-strategy.ts#L31